Automattic/woocommerce-payments
# Incorrect error message when card ZIP validation fails

Fixes #2602

#### Changes proposed in this Pull Request

When an order fails card ZIP code validation, there is an error message telling them to update the ZIP code. Sometimes users update the shipping ZIP code since the error doesn't specify.

<img width="558" alt="127527283-f185ef21-7342-486c-9a41-94523c26c9fa" src="https://user-images.githubusercontent.com/8667118/140037947-7881db15-58ee-45fd-88ef-f0ba749e76fe.png">

With the proposed change the error is changed to `We couldn’t verify the postal code in your billing address. Make sure the information is current with your card issuing bank and try again.` for the notice and `We couldn’t verify the postal code in the billing address. If the issue persists, suggest the customer to reach out to the card issuing bank.` for order notes.

#### Testing instructions

- Open your shop page and add products to the cart.
- Go to the Checkout page and try to pay the order with `4000000000000010` test card.
- Validate the correct error message is displayed on the error notice.
- Validate the correct message is displayed in the Order Notes.

Note:
- If the order is successful [blocking payments that fail postal code validation](https://stripe.com/docs/radar/rules#traditional-bank-checks) needs to be activated for your account.
- Once the above rule is enabled, try again with the `4000000000000010` test card.

-------------------
